packages/twenty-front/src/modules/settings/constants/SettingsObjectModel.ts

```diff
@@ -0,0 +1,2 @@
+export const SETTINGS_OBJECT_MODEL_IS_LABEL_SYNCED_WITH_NAME_LABEL_DEFAULT_VALUE =
+  true;
```

Comment:
Not related to your PR : I saw while investigating, in the same forlder an unused constant EXPANDED_WIDTH_ANIMATION_VARIANTS -> could be removed don't you think ?